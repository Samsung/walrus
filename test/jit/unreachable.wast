(module
(func $f1
  unreachable
)

(func $f2
   call $f1
)

(func $test2 (export "test1")
   call $f2
)
)
