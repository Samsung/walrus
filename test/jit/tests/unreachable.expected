[[[[[[[  Function   0  ]]]]]]]
Number of locals reduced from 0 to 0
0: Opcode: unreachable
------------------------------
Frame size: 0, Arguments size: 0 (total: 0)
0: Opcode: unreachable

[[[[[[[  Function   1  ]]]]]]]
Number of locals reduced from 0 to 0
0: Opcode: call
  Frame size: 0, param start: 0
1: Opcode: return
------------------------------
Frame size: 0, Arguments size: 0 (total: 0)
0: Opcode: call
  Frame size: 0, param start: 0
1: Opcode: return

[[[[[[[  Function   2  ]]]]]]]
Number of locals reduced from 0 to 0
0: Opcode: call
  Frame size: 0, param start: 0
1: Opcode: return
------------------------------
Frame size: 0, Arguments size: 0 (total: 0)
0: Opcode: call
  Frame size: 0, param start: 0
1: Opcode: return

test1() =>
